ios/Runner/AppDelegate.swift

```diff
@@ -56,30 +57,34 @@ import os.log
         // to authenticate. This, the app dying in the background, does not happen on iOS
         let appAuthEvents = FlutterEventChannel(name: APP_AUTH_EVENTS,
                                                 binaryMessenger: controller.binaryMessenger)
-        appAuthEvents.setStreamHandler(StubEventStreamHandler())
+        appAuthEvents.setStreamHandler(StreamHandler())
 
         // TODO receive share events and send them to this channel, see ShareEventStream in Android
         let shareEvents = FlutterEventChannel(name: SHARE_EVENTS,
                                               binaryMessenger: controller.binaryMessenger)
-        shareEvents.setStreamHandler(StubEventStreamHandler())
+        if self.shareStreamHandler == nil {
+            self.shareStreamHandler = StreamHandler()
+        }
+        shareEvents.setStreamHandler(self.shareStreamHandler)
 
         GeneratedPluginRegistrant.register(with: self)
         return super.application(application, didFinishLaunchingWithOptions: launchOptions)
     }
     
-    // Handle redirect from website
     override func application(_ app: UIApplication, open url: URL, options: [UIApplication.OpenURLOptionsKey : Any] = [:]) -> Bool {
         
-        if let authorizationFlow = self.currentAuthorizationFlow, authorizationFlow.resumeExternalUserAgentFlow(with: url) {
-            self.currentAuthorizationFlow = nil
-            return true
-        }
```

Comment:
Why is this no longer needed?

Reply:
I found out that this was never called during auth. (But was in the samples I found)

Reply:
So the field can be removed too?

Reply:
Hm - its still called from AppAuthHandler - it seems to supports the OpenId infrastructure and should be declared in AppDelegate.